core/server/data/meta/description.js

```diff
@@ -14,7 +14,7 @@ function getDescription(data, root) {
     } else if (_.contains(context, 'author') && data.author) {
         description = data.author.bio;
```

Comment:
That behaviour hasn't changed, we've always used bio this way and I believe it is escaped :wink: